src/main/java/com/nike/cerberus/operation/core/UpdateStackOperation.java

```diff
@@ -126,6 +126,7 @@ public void run(UpdateStackCommand command) {
 
     @Override
     public boolean isRunnable(UpdateStackCommand command) {
+        configStore.isConfigSynchronized();
```

Comment:
If there is a region outage, can I still run a command?

Reply:
Also why are you running this in multiple places, could you not do this in a single global place for all commands?